io.github.alainm23.planify.json

```
@@ -0,0 +1,124 @@
+{
+    "app-id" : "io.github.alainm23.planify",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.alainm23.planify",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gnome.evolution.dataserver.Calendar8",
+        "--talk-name=org.gnome.evolution.dataserver.Sources5",
+        "--talk-name=io.github.alainm23.planify"
```

this name is granted by default

```suggestion
        "--talk-name=org.gnome.evolution.dataserver.Sources5"
```